kibana-reports/public/components/main/report_definition_details/report_definition_details.tsx

```diff
@@ -467,17 +467,9 @@ export function ReportDefinitionDetails(props) {
       reportDefinitionRawResponse.report_definition.report_params.core_params
         .time_duration;
     const fromDate = getRelativeStartDate(duration);
```

Comment:
Do we still need all these?

Reply:
No, good catch. Will remove

Reply:
Can you verify if it's the same situation for logic in create_report_defintion.ts? I always have an impression that lots of code can be removed with this new API

Reply:
Do you mean `create_report_definition.tsx`? If so, yes. From the code change in the PR, the only parameter that is created is a `reportDefinitionId` constant based on the response from the `Create` API call